packages/typescript/README.md

```diff
@@ -0,0 +1,33 @@
+# `@metamask/eslint-config-typescript`
+
+MetaMask's [TypeScript](https://www.typescriptlang.org) ESLint configuration.
+
+## Usage
+
+```bash
+yarn add --dev \
+    eslint@^7.7.0 \
+    eslint-plugin-import@^2.22.0 \
+    @typescript-eslint/eslint-plugin@^3.9.1 \
+    @typescript-eslint/parser@^3.9.1 \
+    @metamask/eslint-config-typescript@^5.0.0
+```
+
+```js
+module.exports = {
+  root: true,
+  extends: [
+    '@metamask/eslint-config-typescript',
```

Comment:
We should probably include the base config in all of our examples as well (both the config and the install step).

Reply:
f9e6ed1